internal/acceptance/init_test.go

```diff
@@ -353,6 +354,45 @@ func isInDebug() bool {
 	return strings.HasPrefix(path.Base(ex), "__debug_bin")
 }
 
+func loadWorkspaceEnv(t *testing.T) {
+	loadDebugEnvIfRunsFromIDE(t, "workspace")
+}
+
+func loadAccountEnv(t *testing.T) {
+	loadDebugEnvIfRunsFromIDE(t, "account")
+}
+
+func loadUcwsEnv(t *testing.T) {
+	loadDebugEnvIfRunsFromIDE(t, "ucws")
+}
+
+func loadUcacctEnv(t *testing.T) {
+	loadDebugEnvIfRunsFromIDE(t, "ucacct")
+}
+
+func isAws() bool {
+	awsCloudEnvs := []string{"MWS", "aws", "ucws", "ucacct"}
```

Comment:
Yes, verified by comparing to last night's nightly. There are some small differences due to me fixing test name prefixes, so some test counts are off by one or two compared to the previous nightly run, but otherwise everything lines up.